test/unit/componentHandler.js

```diff
@@ -115,6 +115,14 @@ describe('componentHandler', function() {
     document.body.removeChild(button);
     document.body.removeChild(buttonTwo);
   });
+  
+  it('should upgrade a single component to an element', function() {
```

Comment:
Added test case 'should upgrade a single component to an element' to reveal a bug in the current upgradeElement caused by unreliably checking of upgraded class.